Makefile

```diff
@@ -156,6 +157,12 @@ test/common:
 	git checkout --force FETCH_HEAD -- test/common
 	git reset test/common
 
+SASS = ./node_modules/.bin/sass
+build-patternfly:
+	F=$(srcdir)/lib/patternfly-4-cockpit.scss && \
+	$(SASS) $$F dist/patternfly-4-base.css --source-map --load-path node_modules && \
+	sed -i -f $(srcdir)/lib/patternfly.sed dist/patternfly-4-base.css
```

Comment:
Please let SASS output to $@.tmp, mangle that, and as the very last step `mv $@.tmp $@`. In the case of errors this will make sure that we don't have half-written/broken result files.